mesheryctl/README.md

```diff
@@ -0,0 +1,42 @@
+# mesheryctl
+
+`mesheryctl` is the CLI Client for Meshery.
+
+# Contributing
+
+Please refer the [Meshery Contributing Guidelines](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md) for setting up your development environment.
+
+## Building and running `mesheryctl`
+
+The `~/meshery/mesheryctl` folder contains the complete code for `mesheryctl`.
+
+`mesheryctl` is written in Golang or the Go Programming Language. For development use Go version 1.15+.
+
+After making changes, run `make` in the `mesheryctl` folder to build the binary. You can then use the binary by, say, `./mesheryctl system start`.
+
+## `mesheryctl` command reference
+
+Detailed documentation of the `mesheryctl` commands is available in the [Meshery Docs](https://docs.meshery.io/reference/mesheryctl).
+
+## General guidelines and resources
```

Comment:
Good to add a link to this design spec somewhere in here.

https://docs.google.com/document/d/1xRlFpElRmybJ3WacgPKXgCSiQ2poJl3iCCV1dAalf0k/edit#

Reply:
Added link to CLI design spec